apps/web/src/server/applications/case/project-team/applications-project-team.service.js

```diff
@@ -0,0 +1,35 @@
+import { fixturePaginatedProjectTeamMembers } from '../../../../../testing/applications/fixtures/project-team.js';
+import pino from '../../../lib/logger.js';
+
+/** @typedef {import('@pins/express').ValidationErrors} ValidationErrors */
+/** @typedef {import('../../applications.types').ProjectTeamMember} ProjectTeamMember */
+/** @typedef {import('../../applications.types').PaginatedResponse<ProjectTeamMember>} PaginatedProjectTeamMembers */
+
+/**
+ * Retrieve the paginated list of the team members matching the query
+ *
+ * @param {string} query
+ * @param {number} pageNumber
+ * @returns {Promise<{results: PaginatedProjectTeamMembers, errors: ValidationErrors}>}
+ */
+export const searchProjectTeamMembers = async (query, pageNumber) => {
+	let response;
+
+	try {
+		//response = await get(`applications/${caseId}/key-dates`);
```

Comment:
assume this is not needed?

Reply:
no, because right now the service is just returning mocked data, but I kept it because, once backend is done, it should be uncommented and replaced with the actual endpoint